### ISSUE TITLE
"No such file or directory" when installing the yellow dark variant

### ISSUE THREAD

**Issue by kyechou** [Fausto-Korpsvart/Everforest-GTK-Theme#14]
When I install the yellow dark variant with `./install.sh --theme yellow` or `./install.sh --theme all`, it says

```
Installing '/usr/share/themes/Everforest-Yellow-Dark'...
cp: cannot stat '.../themes/src/assets/gtk/thumbnails/thumbnail-Yellow-Dark.png': No such file or directory
```

**Comment by Fausto-Korpsvart** [comment]
> When I install the yellow dark variant with `./install.sh --theme yellow` or `./install.sh --theme all`, it says
> 
> ```
> Installing '/usr/share/themes/Everforest-Yellow-Dark'...
> cp: cannot stat '.../themes/src/assets/gtk/thumbnails/thumbnail-Yellow-Dark.png': No such file or directory
> ```

@kyechou, Yes, sorry, it was my mistake, I updated the theme, but the assets are not updated yet, I'm working on it, I'll update soon.
